<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>261</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Generate Adaptive Light Source Vectors</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>brl_logo.gif</normaloff>brl_logo.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The top level directory containing all the sessions</string>
    </property>
    <property name="text">
     <string>Focal Length</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The first session to process</string>
    </property>
    <property name="text">
     <string>Sensor Size (x,y)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>101</width>
      <height>13</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The location of the text file giving the regions of interest for the plants</string>
    </property>
    <property name="text">
     <string>PSConfig.properties</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>81</width>
      <height>13</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Where to output the resultant masks</string>
    </property>
    <property name="text">
     <string>LSV Output Dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtFocalLength">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>36</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Focal length of the lens (eg 16 or 25mm)</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtPSConfigFile">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The location of the PSConfig.properties file that contains the ligth source vectors to adapt</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtOutputDir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>351</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Where to output the resultant adaptive light source vectors</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPSConfigSel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOutputSel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>511</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Data settings</string>
    </property>
    <widget class="QLineEdit" name="edtSensorSize">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>56</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The camera's sensor size, PointGrey Grasshopper is 8.8x8.8, rPi is 3.674x2.760</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpBoxLSOverride">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Which light sources to generate the adaptive light source vectors for&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>Light Source Type</string>
     </property>
     <widget class="QRadioButton" name="rdoVIS">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>VIS</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoNIR">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="text">
       <string>NIR</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoVISNIR">
      <property name="geometry">
       <rect>
        <x>103</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>VISNIR</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="edtResolution">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The camera sensor's resolution e.g. 2048x2048 for PointGrey Grasshopper</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The first session to process</string>
     </property>
     <property name="text">
      <string>Resolution (x,y)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The first session to process</string>
     </property>
     <property name="text">
      <string>e.g. 2048x2048, 2592x1944</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The first session to process</string>
     </property>
     <property name="text">
      <string>e.g. 8x8, 3.764x2.7</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnProcess">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
   <zorder>edtFocalLength</zorder>
   <zorder>edtPSConfigFile</zorder>
   <zorder>edtOutputDir</zorder>
   <zorder>btnPSConfigSel</zorder>
   <zorder>btnOutputSel</zorder>
   <zorder>btnProcess</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>edtFocalLength</tabstop>
  <tabstop>edtSensorSize</tabstop>
  <tabstop>edtResolution</tabstop>
  <tabstop>btnPSConfigSel</tabstop>
  <tabstop>btnOutputSel</tabstop>
  <tabstop>rdoVIS</tabstop>
  <tabstop>rdoNIR</tabstop>
  <tabstop>rdoVISNIR</tabstop>
  <tabstop>btnProcess</tabstop>
  <tabstop>edtPSConfigFile</tabstop>
  <tabstop>edtOutputDir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
